src/components/ContactInfo/ContactInfo.vue

```diff
@@ -186,7 +186,7 @@
             >
             </v-text-field>
           </v-col>
-          <v-col>
+          <v-col class="ml-2">
```

Comment:
Won't this cause it to not line up with other columns on the page?

Reply:
This is the split between the two input fields. The cols are on auto, so splitting the margin between the two elements keeps it equal. 

![Screen Shot 2022-03-04 at 10 01 02 AM](https://user-images.githubusercontent.com/53542131/156817140-ff20aa65-d426-47a6-b815-ead5e51b422b.png)

Reply:
Oh those fields. I thought this was changing the main columns 1 and 2.

Reply:
Updated the approach to use just padding on the Col 1 and let the auto cols handle the second. 
Works the same, looks the same. Styling in 1 place though 👍

Reply:
When the phone/extension columns wrap, is the second column indented due to the left margin?

Reply:
Oh nvm -- I see you latest code has only right margin on the first column, so the second column should be fine.